experiments/test_all.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
No need to actually rename the files, but I would not call these test as they are more like runners or helpers.